quax/_core.py

```diff
@@ -537,4 +537,34 @@ def _(*args: Union[ArrayLike, ArrayValue], jaxpr, inline, **kwargs):
             return jax.jit(flat_fun)(leaves)  # now we can call without Quax.
 
 
+@register(jax.lax.while_p)
+def _(
+    *args: ArrayValue | ArrayLike,
```

Comment:
I think this is only Python 3.10+? For now we're on 3.9+.

Reply:
That is true, i locally used Python 3.11 and good this mixed up. It is changed now.